lang/java/reef-tang/tang-test-jarA/src/main/java/org/apache/reef/tang/examples/A.java

```diff
@@ -19,5 +19,4 @@
 package org.apache.reef.tang.examples;
 
 public class A {
-  private int iAmAClass = 0;
```

Comment:
It is an unused field, and as far as I understand the field itself is not needed for the example (it's never mentioned in any code), it's the classes that are needed. In jarAB test the class has no fields.